### DISCUSSION TITLE
Update on the Statiq Project, Pending PRs and Issues, etc.

### DISCUSSION THREAD

**Discussion by daveaglick** [statiqdev/Discussions#197] [General]
Hi all! Don't worry, this isn't an "I'm archiving the project" note 😬. I just figured it was long overdue to give an update on why it might seem that way right now.

As many of you have observed, there's been an extreme delay in my ability to answer issues, review and merge PRs, and generally maintain the project. The main reason is that I've been slowly closing in on the culmination of a large, multi-year effort at my day job. This, combined with the increased parental responsibilities that come with your children getting older, has meant a lack of time for open source. For better or worse, when weighed against the job that pays my bills, my family, my community, and all the rest, open source comes pretty close to the bottom of the queue.

That's not to say I'm abandoning the project. Quite the opposite - I literally use it all day every day at my day job and it powers a large portion of our software infrastructure (including as our management and deployment tool for Azure, I really need to do a write-up on that one day). It's also still my intent to ramp up community engagement and activity here as soon as I'm able, though I'll be honest and say I have no idea when that might be, and whether it's measured in weeks or months. In the meantime, I'll still drop in as often as I'm able and clear what I can.

Hope everyone else is doing well, and thanks for being a part of my (not so?) little project!

**Comment by harperjohnhome** [comment]
I just stumbled on this project - and instantly got very excited about! Glad to hear that you plan on keeping it going!

Hopefully I can become a contributor!

**Reply by daveaglick** [reply]
Yay, I'm excited that you're excited! :)

Feel free to open any discussions here about questions or idea you may have. Like I said above, I may not get to them immediately, but I'm trying hard to make sure I get to everything new in at least a reasonable timeframe.

**Reply by harperjohn** [reply]
Really glad to hear that you are excited about my excitement!! :)
